index.js

```diff
@@ -47,6 +47,20 @@ const worker = queue.async.asyncify(doWork => doWork());
 
 module.exports = Filter;
 
+module.exports.shouldPersist = shouldPersist;
+
+function shouldPersist(env, persist) {
+  let result;
+
+  if (env.CI) {
```

Comment:
@stefanpenner it might make sense to use something like https://github.com/watson/ci-info here instead

Reply:
Yes, agreed. I had made an issue over on ember-cli to work on migrating things to a shared common definition of “I’m in a CI environment”...